doc/cookbook/source/examples/classifier/qda.rst

```diff
@@ -0,0 +1,39 @@
+===============================
+Quadratic discriminant analysis
```

Comment:
btw can you rename all the files to their long name?
Sorry, but this is something that just came up in chat. Includes the data files (unfortunately)